locales/en/public.json

```diff
@@ -363,8 +363,8 @@
       "TITLE": "Theme"
     },
     "WEBSOCKET_CONNECTION_DEBOUNCE": {
-      "DESCRIPTION": "Set the debounce time (in milliseconds) used when establishing WebSocket connections. Increase this time if the web-interface repeatedly displays WebSocket connection/disconnection messages. Decrease this time if the web-interface takes a long time to populate on startup.",
-      "TITLE": "WebSocket Connection Debounce"
+      "DESCRIPTION": "Set the retry interval (debounce) time (in milliseconds) used when establishing WebSocket connections. Increase this time if the web-interface repeatedly displays WebSocket connection/disconnection messages. Decrease this time if the web-interface takes a long time to populate on startup.",
```

Comment:
Remove "...(debounce) time..."?

Reply:
Yes, I kept this so one can relate if they read the code because internally everywhere it is named as debounce.

Reply:
Yeh but this will be display for the users. So, we need to move them. Don't worry, internally it should be easy enough to recognize by contributors.